### PR TITLE
Remove references rush.json references in the scripts (aside from create-package for now)

### PR DESCRIPTION
**Pull request by kenotron**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

We are moving away from Rush references in the script. The scripts affected:

1. lint-imports
2. prettier
3. update-package-versions
4. generate-versions

Also, I noticed that set-version script was busted. So I fixed it with this PR as well!

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9817)

### CODE REVIEW

File: scripts/monorepo/getAllPackageInfo.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const { spawnSync } = require('child_process');
+const { readConfig } = require('../read-config');
+const path = require('path');
+const findGitRoot = require('./findGitRoot');
+
+module.exports = function getAllPackageInfo() {
+  const gitRoot = findGitRoot();
+  const results = spawnSync('git', ['ls-tree', '-r', '--name-only', '--full-tree', 'HEAD']);
```
Comment (jdhuntington) [inline scripts/monorepo/getAllPackageInfo.js]: I think it's marginally faster to run `git ls-files '*/package.json'`, but has the benefit of not needing to filter out entries after retrieving them.

Comment (kenotron) [reply scripts/monorepo/getAllPackageInfo.js]: oh cool tip!

Comment (kenotron) [reply scripts/monorepo/getAllPackageInfo.js]: i'll fix it next go around maybe :)

File: scripts/tasks/lint-imports.js [hunk 2]
```diff
@@ -247,14 +248,17 @@ module.exports = function() {
 
       const pkgName = pkgNameMatch[1] || pkgNameMatch[2];
 
-      // we don't evaluate imports of non rush packages
-      if (rushPackages.indexOf(pkgName) === -1) return;
+      // we don't evaluate imports of non monorepo packages
+      if (!Object.keys(packagesInfo).includes(pkgName)) {
```
Comment (ecraig12345) [inline scripts/tasks/lint-imports.js]: Would be a lot more efficient to do `!packagesInfo[packageName]`

File: scripts/update-package-versions.js [hunk 3]
```diff
@@ -34,17 +34,17 @@ if (!newVersion || !newDep) {
...
 
-  console.log(`Updating ${chalk.magenta(package.packageName)} from ${chalk.grey(packageJson.version)} to ${chalk.green(newVersion)}.`);
+  console.log(`Updating ${chalk.magenta(name)} from ${chalk.grey(packageJson.version)} to ${chalk.green(newVersion)}.`);
 
   packageJson.version = newVersion;
 
   function updateDependencies(deps) {
     for (const dependency in deps) {
-      if (rushJson.projects.find(pkg => pkg.packageName === dependency)) {
+      if (Object.keys(allPackages).find(name => name === dependency)) {
```
Comment (ecraig12345) [inline scripts/update-package-versions.js]: Can't this just be `if (allPackages[dependency])` now?

**Pull request merged**